### DISCUSSION TITLE
Thank you for your work! ♥️

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [DuCanhGH/next-pwa#117] [General]
Thank you so much @DuCanhGH, that you forked next-pwa and are developing it further! I'm so happy that you are doing it, and I was able to convert my site (https://weatherio1.vercel.app) into a PWA!

♥️♥️♥️

**Comment by DuCanhGH** [comment]
Thanks a lot for the compliment! That means a lot to me :) It's nice that you got the app up and working!
